modules/displayTime.js

```diff
@@ -0,0 +1,10 @@
+import { DateTime } from './luxon.min.js';
+
+// DISPLAY CURRENT TIME LOGIC
+export default function displayTime() {
```

Comment:
- [ ]  I will encourage you to change all functions to the arrow function so that it would help javascript in the runtime and avoid hoisting.